Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -138,6 +138,10 @@
     <string name="slot_a_device">GameCube Slot A Device</string>
     <string name="slot_b_device">GameCube Slot B Device</string>
     <string name="wii_submenu">Wii</string>
+    <string name="insert_SD_Card">Insert SD Card</string>
+    <string name="insert_SD_Card_description">Default size is 128 MB. Supports SDSC and SDHC.</string>
```

Comment:
SDSC is not an acronym I think I've seen before. You may want to say "SD and SDHC" instead.